active_versioning.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rails", "~> 5.0"
+  spec.add_dependency "rails", ">= 5.0"
```

Comment:
The previous syntax used a unqualified ">" which would cause an external dependancy with rails 6 to fail. switching `>`  to `>=` resolves this issue for us